node/src/main/scala/coop/rchain/node/api/ReplGrpcService.scala

```diff
@@ -0,0 +1,64 @@
+package coop.rchain.node.api
+
+import coop.rchain.node.diagnostics
+import coop.rchain.p2p.effects._
+import io.grpc.{Server, ServerBuilder}
+
+import scala.concurrent.Future
+import cats._, cats.data._, cats.implicits._
+import com.google.protobuf.empty.Empty
+import coop.rchain.casper.{MultiParentCasper, PrettyPrinter}
```

Comment:
Seems like there are a lot of unused imports here, should probably be cleaned up.

Reply:
I'm planning to provide a automated scala-fix for that next sprint. Don't want to do it manually :)

Reply:
Cool :)